kiwi/package_manager/zypper.py

```diff
@@ -149,11 +157,15 @@ def update(self):
         chroot_zypper_args = self.root_bind.move_to_root(
             self.zypper_args
         )
+        chroot_command_env = self.command_env
+        chroot_command_env['ZYPP_CONF'] = self.root_bind.move_path_to_root(
+            self.command_env['ZYPP_CONF']
+        )
```

Comment:
This should not be needed, ZYPP_CONF should already be part of the environment because the PackageManager instance get this information from self.repository.runtime_config() in the post_init constructor